tests/MSBuild/MSBuildWorkspaceLoaderTests.cs

```diff
@@ -30,26 +37,31 @@ public MSBuildWorkspaceLoaderTests(ITestOutputHelper output)
         [MSBuildTheory(typeof(WindowsOnly))]
         [InlineData("winforms")]
         [InlineData("winformslib")]
-        // Skip="https://github.com/dotnet/format/issues/1402"
-        // [InlineData("wpf")]
-        // [InlineData("wpfusercontrollib")]
-        // [InlineData("wpflib")]
-        // [InlineData("wpfcustomcontrollib")]
-        public async Task CSharpTemplateProject_WindowsOnly_LoadWithNoDiagnostics(string templateName, string ignoredDiagnostics = "")
+        [InlineData("wpf")]
+        [InlineData("wpfusercontrollib")]
+        [InlineData("wpflib")]
+        [InlineData("wpfcustomcontrollib")]
+        public async Task CSharpTemplateProject_WindowsOnly_LoadWithNoDiagnostics(string templateName)
         {
-            await AssertTemplateProjectLoadsCleanlyAsync(templateName, LanguageNames.CSharp, ignoredDiagnostics.Split(";", StringSplitOptions.TrimEntries));
+            var ignoredDiagnostics = templateName switch
```

Comment:
Sad that these diagnostics mean that MSBuildWorkspace isn't able to load those project types correctly.

Reply:
I imagine it would be able to if it ran additional build targets to generate the output files for the XAML content.